doc/GSG/hybrid_cpu_support.rst

```diff
@@ -0,0 +1,38 @@
+.. _hybrid_cpu_support:
+
+Hybrid CPU and NUMA Support
+***************************
+
+If you need NUMA/Hybrid CPU support in oneTBB, you need to make sure that HWLOC* is installed on your system.
+
+HWLOC* (Hardware Locality) is a library that provides a portable abstraction of NUMA and hybrid CPU systems. 
```

Comment:
Can we reuse a definition from the official website?
```suggestion
HWLOC* (Hardware Locality) is a library that provides a portable abstraction of the hierarchical topology of modern architectures (NUMA, hybrid CPU systems, etc). 
```
What do you think?

Reply:
Updated.